lexer/token.go

```diff
@@ -12,6 +12,7 @@ const (
 	CONCAT // Not really a token, but used as an operator
 
 	// Symbols
+
```

Comment:
this extra line prevents the section comment from being mistaken as documentation for `ADD`